Yoast/YoastSEO.js
# Rename QueryStringAppender back to Shortlinker

## Summary

This PR can be summarized in the following changelog entry:

* Added functionality to append a query string to the assessment links through the analysis worker.

## Relevant technical choices:

* Revert rename because it is confusing if it is not called the same as in the plugin.
* Changed the import in the `FunctionWordsInKeyphraseAssessment`. As this assessment was added in the meantime.

## Test instructions

This PR can be tested by following these steps:

* See: https://github.com/Yoast/wordpress-seo/pull/11330

Fixes #1889
